docker/yb-voyager-docker

```diff
@@ -54,6 +54,12 @@ for var_name in "${variables[@]}"; do
   fi
 done
 
+flags_to_store=("--source-ssl-cert"
+"--source-ssl-key"
+"--source-ssl-root-cert")
+
+file_name="/tmp/voyager-docker-metadata.txt"
```

Comment:
This name should be export-dir/migrationUUID specific. Otherwise, it will interfere with other migrations running on the same host.

Reply:
Consider moving the temp file into the export-dir once you have collected all the arguments. That way it will reside inside the export-dir.